protocol_rfcs/iceberg-writer-compat-v1.md

```diff
@@ -0,0 +1,93 @@
+# IcebergWriterCompatV1
+**Associated Github issue for discussions: https://github.com/delta-io/delta/issues/4284
+
+This protocol change introduces a compatibility flag, which ensures that a delta table can be safely
+read and written as an Apache Iceberg™ format table, similar to
+[IcebergCompatV1](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#iceberg-compatibility-v1)
+and
+[IcebergCompatV2](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#iceberg-compatibility-v2).
+
+--------
+
+# IcebergWriterCompatV1
+> ***New Section after [Iceberg Compatibility V2](#iceberg-compatibility-v2)***
+
+This table feature (`icebergWriterCompatV1`) ensures that Delta tables can be converted to Apache
+Iceberg™ format, though this table feature does not implement or specify that conversion.
+
+To support this feature:
+- Since this table feature depends on Column Mapping, the table must be on Reader Version = 2, or it must be on Reader Version >= 3 and the feature `columnMapping` must exist in the `protocol`'s `readerFeatures`.
```

Comment:
Not an expert in this but isn't "Reader Version = 3" implicitly means `columnMapping` is enabled?

Reply:
Actually not. For reader 3 / writer 7 the features _must_ be present in the reader/writer features in the `protocol`. Generally when updating from an earlier version that implicitly supports a feature, the engine will ensure that it writes a protocol that includes those features.